protocol_details/asymmetric_mp.rst

```diff
@@ -8,11 +8,11 @@ An embedded AMP system is characterized by multiple homogeneous and/or heterogen
 
     - The Xilinx MPSoC that has four ARM Cortex-A53, two ARM Cortex-R5, and potentially a number of MicroBlaze cores.
     - The NXP i.MX6SoloX/i.MX7d SoCs that utilizes ARM Cortex-A9 and ARM Cortex-M4F cores
-    - The Texas Instruments TI AM57x SoCs that have dual ARM Cortex A15, dual ARM Cortex M4, and C66x DSP cores.
+    - The Texas Instruments TI AM625 SoCs that contain ARM Cortex-A53, ARM Cortex-M4F, ARM Cortex-R5F, and PRU-ICSS cores.
```

Comment:
1) @tnmysh : does we replace Xilinx by AMD?
2) contain or embed or have ? 
   to use same verb for NXP and XilinX Socs
 
3) Could you add missing ST Socs ( else I will do it in a separate PR)?
-  The ST stm32MP15x SoCs that embeded/contain/have ARM Cortex-A7 and ARM Cortex-M4 cores.

Reply:
@arnopo Yes I had same concern. We use AMD-Xilinx to refer old IP.

Reply:
But it's a nitpick. I don't see it as a blocker to merge this one. Lot of other place this change is needed.

Reply:
Updated `Xilinx MPSoC` to `AMD Zynq UltraScale+ MPSoC` which seems to match your current branding, let me know if I didn't get that right.

Added the ST example (makes sense to have an example for each vendor with supported parts).

I've switched to using "utilize" to be consistent in all examples.

Reply:
Above works. Thanks.